include/dlaf/matrix/tile.h

```diff
@@ -34,12 +34,22 @@ struct ContinuationException final : public std::runtime_error {
 };
 
 namespace matrix {
+// forward declarations
 template <class T, Device device>
 class Tile;
 
 template <class T, Device device>
 class Tile<const T, device>;
 
+template <class T, Device D>
+std::vector<hpx::shared_future<Tile<const T, D>>> splitTile(
+    hpx::shared_future<Tile<const T, D>>& tile,
+    std::vector<std::tuple<TileElementIndex, TileElementSize>> specs);
```

Comment:
Is it maybe worth creating a specific type for `tuple<TileElementIndex, TileElementSize>`?